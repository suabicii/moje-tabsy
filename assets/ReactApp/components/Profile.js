import React from "react";

function Profile() {
    return (
        <>
            <h1 className="text-center mt-5 mt-md-0">MÃ³j profil</h1>
        </>
    );
}

export default Profile;