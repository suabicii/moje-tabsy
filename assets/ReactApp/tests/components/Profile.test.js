import React from "react";
import ReactShallowRenderer from "react-test-renderer/shallow";
import Profile from "../../components/Profile";

it('should correctly render Profile page', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<Profile fakeApiUrl="/fake-api/user-data"/>);

    expect(renderer.getRenderOutput()).toMatchSnapshot();
});