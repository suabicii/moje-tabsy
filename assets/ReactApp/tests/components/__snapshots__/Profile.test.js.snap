// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render Profile page 1`] = `
<React.Fragment>
  <h1
    className="text-center mt-5 mt-md-0"
  >
    MÃ³j profil
  </h1>
  <ProfileForm
    fetchData={Promise {}}
  />
</React.Fragment>
`;
